### PR TITLE
Adding unit tests

### PR DESCRIPTION
**Pull request by martojdk**

### CODE REVIEW

File: src/com/service/CreatePersonService.java [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+                    System.out.println(USE_INTEGERS_ONLY + e.getMessage());
+                }
+            } else {
+                wrongInputByUserReturnNewLine();
+                System.out.println(USE_INTEGERS_ONLY);
+            }
+        }
+    }
+
+    public static Date formatDateOfBirth(String dateOfBirthString) {
```
Comment (martojdk) [inline src/com/service/CreatePersonService.java]: This should be in a separate class named utils. I will explain this

File: src/com/service/CreatePersonService.java [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+        }
+        return dateOfBirth;
+    }
+
+    public static boolean isValidDateOfBirth(String dateOfBirth) {
+        if(dateOfBirth == null || !dateOfBirth.matches("\\d{4}-[01]\\d-[0-3]\\d")) {
+            return false;
+        }
+        SimpleDateFormat dateFormat = new SimpleDateFormat("yyyy-MM-dd");
+        dateFormat.setLenient(false);
```
Comment (martojdk) [inline src/com/service/CreatePersonService.java]: Is that really needed?

File: src/com/service/CreatePersonService.java [hunk 3]
```diff
@@ -0,0 +1,178 @@
...
+            } catch (ParseException e) {
+                e.printStackTrace();
+                System.out.println(USE_INTEGERS_ONLY);
+            }
+        }
+        return dateOfBirth;
+    }
+
+    public static boolean isValidDateOfBirth(String dateOfBirth) {
+        if(dateOfBirth == null || !dateOfBirth.matches("\\d{4}-[01]\\d-[0-3]\\d")) {
```
Comment (martojdk) [inline src/com/service/CreatePersonService.java]: Put these strings in constants

File: src/com/service/PersonService.java [hunk 4]
```diff
@@ -34,7 +33,7 @@ private boolean isPersonCreated(Person person) {
 
     @Override
     public boolean addNewPerson(Person person) {
-        if(isPersonCreated(person)) {
+        if (isPersonCreated(person) || person == null) {
```
Comment (martojdk) [inline src/com/service/PersonService.java]: Maybe person== null check should be in a separate if condition. If the user passes null he gets the message that person exists which is not true.

File: src/com/service/PersonServiceTest.java [hunk 5]
```diff
@@ -10,42 +10,79 @@
...
+        Person person = new Person("", EGender.MALE, new Date(), "dsfsd", EStatus.IN_RELATIONSHIP);
+    }
+
+    @org.junit.jupiter.api.Test
+    public void givenNullablePerson_whenCreatingPerson_thenExpectNotToBeAddedInList() {
+        assertFalse(classUnderTest.addNewPerson(null));
     }
 
     @org.junit.jupiter.api.Test
     void removePerson() {
```
Comment (martojdk) [inline src/com/service/PersonServiceTest.java]: given_when_then.. apply it to the other methods and don't forget to stay consistent with your code

File: src/com/service/PersonService.java (hunk 4 above)
Comment (kostana1) [reply src/com/service/PersonService.java]: I've seen this when testing, but forgot to fix it

File: src/com/service/CreatePersonService.java (hunk 3 above)
Comment (kostana1) [reply src/com/service/CreatePersonService.java]: Sorted

File: src/com/service/CreatePersonService.java (hunk 2 above)
Comment (kostana1) [reply src/com/service/CreatePersonService.java]: Nope, just still noob here

File: src/com/service/CreatePersonService.java (hunk 1 above)
Comment (kostana1) [reply src/com/service/CreatePersonService.java]: Sorted

**Pull request merged**